todo_app/data/trello_items.py

```diff
@@ -0,0 +1,35 @@
+import os, requests
```

Comment:
You seem to have updated your poetry.lock file with requests but not the actual list of dependencies in pyproject.toml. This means other people won't end up with the requests library after running "poetry install".

Not sure how you managed that but you should have run `poetry add requests` from your terminal, which would update pyproject.toml for you and download requests. Maybe you just didn't commit the change to pyproject.toml?